web/src/main/webapp/WEB-INF/pages/candidate_create.jsp

```diff
@@ -198,6 +204,71 @@
                 </div>
             </div>
         </div>
+
+        <c:if test="${cv_status eq true}">
+        <c:set var="candidateCvPath" value="${candidate.cvPath}" />
+        <c:set var="cvPathArray" value="${fn:split(candidateCvPath, '\\\\')}" />
+        <c:set var="cvName" value="${cvPathArray[fn:length(cvPathArray)-1]}" />
```

Comment:
CV name should be added by the controller to the model. We should not do this logic in the view.